packages/remark-math/package.json

```diff
@@ -28,8 +28,13 @@
     "trim-trailing-lines": "^1.1.0"
   },
   "files": [
-    "index.js",
-    "block.js",
-    "inline.js"
-  ]
+    "dist"
```

Comment:
Maybe it's better to keep the explicit file list, but I figure that's in the build command now. Not sure if there's something better that people typically do.